src/wallet/rpcwallet.cpp

```diff
@@ -3989,7 +3998,7 @@ static const CRPCCommand commands[] =
     { "wallet",             "encryptwallet",                    &encryptwallet,                 {"passphrase"} },
     { "wallet",             "getaddressesbylabel",              &getaddressesbylabel,           {"label"} },
     { "wallet",             "getaddressinfo",                   &getaddressinfo,                {"address"} },
-    { "wallet",             "getbalance",                       &getbalance,                    {"dummy","minconf","include_watchonly"} },
+    { "wallet",             "getbalance",                       &getbalance,                    {"trusted_only|account","minconf","include_watchonly"} },
```

Comment:
Should keep this as `trusted_only|dummy` to not break existing clients that use named args?

Reply:
Hmm. Probably not, since afaik "dummy" has only ever been used in versions where it's actually broken anyway?

Reply:
Removing `account` appears to have been a bug, since we still accept `"*"` as a value?

Reply:
@luke-jr only as a dummy, hence the name, accounts are completely removed

Reply:
Existing code calling `getbalance(account="*")` shouldn't be broken...

Reply:
@luke-jr yes it should in 0.18 and in 0.17 without explicitly enabling deprecated accounts?

Reply:
"*" isn't an account.

Reply:
Then why would specifying it using `account=` make sense?

Reply:
Well, that name was already removed, so NACK on adding it back again

Reply:
Removing it introduced a bug by breaking existing code that uses the `"*"` mode.